src/lib.rs

```diff
@@ -144,7 +164,8 @@ pub fn generate_implib_for_target(out_dir: &Path, arch: &str, env: &str) -> Resu
     };
 
     // Run the selected `dlltool` executable to generate the import library.
-    let status = build_dlltool_command(command, dlltool, arch, &defpath, out_dir).status()?;
+    let status =
+        build_dlltool_command(command, dlltool, &implib_file, arch, &defpath, out_dir).status()?;
```

Comment:
Maybe it's time to define a struct like `DllToolCommand` or something?
A free function with 6 different arguments is a bit too much...
This could be
```
impl DllToolCommand {
    fn build(&self) -> Command {
        ...
    }
}
```

Reply:
I still think this approach is worth trying, but there's no need to address it in this PR.
Maybe I'll attempt to do it myself when I have the time.